### PR TITLE
Add repeated capabilities

### PR DESCRIPTION
**Pull request by ni-erikcrank**
- [ ] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/nimi-python/blob/master/CONTRIBUTING.md).

TODO: Check the above box with an 'x' indicating you've read and followed [CONTRIBUTING.md](https://github.com/ni/nimi-python/blob/master/CONTRIBUTING.md).

- [ ] I've updated [CHANGELOG.md](https://github.com/ni/nimi-python/blob/master/CHANGELOG.md) if applicable.

TODO: Check the above box with an 'x' if considered if there were any and then documented client facing changes in [CHANGELOG.md](https://github.com/ni/nimi-python/blob/master/CHANGELOG.md). Strike through if this is not a relevant client facing change.

- [ ] I've added tests applicable for this pull request

TODO: Check the above box with an 'x' indicating you have considered and added any applicable system or unit tests. Strike through if you considered and decided additional tests were not warranted.

### What does this Pull Request accomplish?

TODO: Include high-level description of the changes in this pull request.

### List issues fixed by this Pull Request below, if any.

TODO: List of issues.

* Fix #xxx
* Fix #yyy

### What testing has been done?

TODO: Detail what testing has been done to ensure this submission meets requirements.

### CODE REVIEW

**Comment by sbethur** [comment]
ok to test

File: generated/nidcpower/nidcpower/session.py [hunk 1]
```diff
@@ -326,12 +326,6 @@ class _SessionBase(object):
     power source to generate power. Use the power_source_in_use property to retrieve this information.
 
     Note: This property does not necessarily indicate if the device is using the auxiliary
-
```
Comment (ni-erikcrank) [inline generated/nidcpower/nidcpower/session.py:329]: This deletion is not the result of my changes. Perforce changelist 49190802 caused this deletion.

File: generated/nidcpower/nidcpower/session.py [hunk 2]
```diff
@@ -1220,12 +1389,6 @@ class _SessionBase(object):
     '''Type: enums.PowerSourceInUse
 
     Indicates whether the device is using the internal or auxiliary power source to generate power.
-
-    Tip:
-    This property can use repeated capabilities. If set or get directly on the
-    nidcpower.Session object, then the set/get will use all repeated capabilities in the session.
-    You can specify a subset of repeated capabilities using the Python index notation on an
-    nidcpower.Session repeated capabilities container, and calling set/get value on the result.
```
Comment (ni-erikcrank) [inline generated/nidcpower/nidcpower/session.py:1228]: Not my change. From Perforce CL 49190802

File: generated/nidcpower/nidcpower/session.py [hunk 3]
```diff
@@ -2684,11 +2979,11 @@ def configure_aperture_time(self, aperture_time, units=enums.ApertureTimeUnits.S
             units (enums.ApertureTimeUnits): Specifies the units for **apertureTime**.
                 **Defined Values**:
 
-                +--------------------------------------------+------------------------------+
-                | ApertureTimeUnits.SECONDS (1028)           | Specifies seconds.           |
-                +--------------------------------------------+------------------------------+
-                | ApertureTimeUnits.POWER_LINE_CYCLES (1029) | Specifies Power Line Cycles. |
-                +--------------------------------------------+------------------------------+
```
Comment (ni-erikcrank) [inline generated/nidcpower/nidcpower/session.py:2691]: Not my change.

**Pull request closed without merging**